isis/scripts/downloadIsisData

```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
-        rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+        args = create_rclone_arguments(dest, "legacybase_usgs:", ntransfers, kwargs)
+        rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
     elif(mission.upper() == "ALL"):
         supported_missions.pop("legacybase")
         for mission, remotes in supported_missions.items():
             for remote in remotes:
-                args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-                rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+                args = create_rclone_arguments(dest, remote, ntransfers, kwargs)
+                rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
     else:
         for remote in supported_missions[mission]:
-            args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-            rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+            args = create_rclone_arguments(dest, remote,  ntransfers, kwargs)
+            rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
 
 
 if __name__ == '__main__':
+    usageString = "usage: downloadIsisData mission dest [-h] [-v] [-n NUM_TRANSFERS] [--config CONFIG] [rclone flags]"
     helpString = (
-    '''This will allow for a user to download isis data directly to their machine from the USGS S3 buckets as well as public end points\n\n'''
-    '''<Mission name> <destination to copy to> <--dry-run (optional)> \n\n'''
-    '''Example of how to run this program:\n\n'''
-    '''    downloadIsisData tgo ~/isisData/\n\n'''
-    '''NOTE: if you would like to download the data for every mission supply the value ALL for the <Mission name> parameter''')
-    parser = argparse.ArgumentParser(description = helpString, formatter_class=argparse.RawTextHelpFormatter)
-    parser.add_argument('mission', help='mission for files to be downloaded')
+    '''
+    Use to copy ISIS data from USGS sources as well as public end points. 
+
+    This script combines multiple sources using the rclone command under the hood. 
+    
+    After positional args, accepts arbitrary rclone flags. Run "rclone help flags" to get a full list of rclone flags. 
+    
+    See the ISIS repo (https://github.com/USGS-Astrogeology/ISIS3) for more info. 
+    
+    Examples of how to run this program:
+        
+        # download everything for tgo
+        $ downloadIsisData tgo $ISISDATA
+    
+        # download everythin except kernels with debug output
+        $ downloadIsisData all $ISISDATA --dry-run --exclude="kernels/**" -vv
+        
+        # download only cks and fks
+        $ downloadIsisData all $ISISDATA --include="{ck/**,fk/**}"''')
```

Comment:
On the off change someone copies this line from the source, maybe push the `''')` to a newline?